modular_skyrat/modules/nanotrasen_rep/code/clothing.dm

```diff
@@ -20,7 +20,7 @@
 	desc = "It's a green jumpsuit with some gold markings denoting the rank of \"Nanotrasen Consultant\"."
 	name = "nanotrasen consultant's jumpsuit"
 	icon_state = "nt_consultant"
-	inhand_icon_state = "b_suit"
+	inhand_icon_state = "dg_suit"
```

Comment:
Not a fix but more a 'why the hell is the inhand for this a blue jumpsuit and not the green+goldtrim one'